src/App.tsx

```diff
@@ -123,15 +123,17 @@ const App: React.FC = () => {
               <Farm title={'Index - Liquidity Mining'} />
             </Route>
             <Route
-              exact
-              path='/join'
+              exact path='/join'
               render={() => (window.location.href = discordLink)}
             />
             <Route
-              exact
-              path='/discord'
+              exact path='/discord'
               render={() => (window.location.href = discordLink)}
             />
+            <Route
```

Comment:
I think this may be too complicated. Try just wrapping the logo image in a React Link component like 
```typescript
<Link to="websiteLink">
    <StyledLogo>...
</Link>
```

Reply:
Fair...lol. That makes much more sense. While I am in there, do we still need the paths routed to `/join` and `/discord` or no?